ngx-fudis/projects/ngx-fudis/src/lib/components/breadcrumbs/breadcrumbs.component.html

```diff
@@ -0,0 +1,11 @@
+<nav aria-label="Breadcrumb navigation" class="fudis-breadcrumbs" *ngIf="links.length > 0">
+	<ng-container *ngFor="let link of links; let last = last">
+		<fudis-link
```

Comment:
Fix the semantic structure, links should be wrapped inside list

`<ul><li><fudis-link /></li></ul>`


Also, current page should be communicated for screen readers with `aria-current="page"`.